src/explorer/resultsList.ts

```diff
@@ -177,7 +179,7 @@ export class ResultsList {
             // Collapse All button
             const collapseAllButton: HTMLSpanElement = <HTMLSpanElement>expandAllButton.cloneNode();
             collapseAllButton.textContent = "-";
-            collapseAllButton.title = "Collapse all groups";
+            collapseAllButton.title = "Collapse all s";
```

Comment:
> s [](start = 51, length = 2)

s?

Reply:
hmm, oops

---
In reply to: [413106683](https://github.com/microsoft/sarif-vscode-extension/pull/254#discussion_r413106683) [](ancestors = 413106683)